client/verta/verta/runtime.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+"""Classes and functions to enable context logging within predictions made against
+models on the Verta Platform.
+
+.. versionadded:: 0.22.0
+
+"""
+
+import json
+import threading
+from typing import Any, Dict, Optional
+
+
+_THREAD = threading.local()
+
+def _init_thread_logs() -> None:
+    """
+    Initialize our thread-local variable for storing logging context.
+    """
+    if not hasattr(_THREAD, 'logs'):
+        _THREAD.logs = dict()
+
+
+def _get_thread_logs() -> Dict[str, Any]:
+    """
+    Return the current thread-local logs or initialize a new dict.
+    """
+    _init_thread_logs()
+    return _THREAD.logs
+
+
+def _set_thread_logs(logs: Dict[str, Any]) -> Dict[str, Any]:
+    """
+    Sets the thread-local logs, overwriting any existing values.
+    """
+    _THREAD.logs = logs
+    return _THREAD.logs
+
+
+def _init_validate_flag() -> None:
+    """
+    Initialize our thread-local variable for the validation flag.
+    """
+    if not hasattr(_THREAD, 'validate'):
+        _THREAD.validate = False
+
+
+def _get_validate_flag() -> bool:
+    """
+    Return the current thread-local variable for validate or initialize a
+    new boolean.
+    """
+    _init_validate_flag()
+    return _THREAD.validate
+
+
+def _set_validate_flag(flag: bool) -> bool:
+    """
+    Sets the thread-local validate flag boolean value.
+    """
+    _THREAD.validate = flag
+    return _THREAD.validate
+
+
+def validate_json(value: Any) -> str:
```

Comment:
If this is just for our internal use, we should mark it as such

```suggestion
def _validate_json(value: Any) -> str:
```

Otherwise we should expand the docstring.

Reply:
Done